src/actions/userActions.js

```diff
@@ -16,6 +16,7 @@ export const login = user => async dispatch => {
       data: { user: createdUser }
     } = await userService.login({ user });
     dispatch(saveUser(createdUser));
+    dispatch(saveSession());
```

Comment:
This doesn't need a saveSession here. The session is saved when it comes with the headers
https://github.com/rootstrap/open-form-admin-webapp/blob/master/src/api/utils/applyDefaultInterceptors.js#L40

Reply:
Removed :)